### PR TITLE
Add io.bit3.ThreemaQT

### PR DESCRIPTION
**Pull request by tristanlins**
ThreemaQT is a qt based desktop app for [Threema Web](https://web.threema.ch/).

### CODE REVIEW

**Comment by tristanlins** [comment]
bot, build io.bit3.ThreemaQT

**Comment by tristanlins** [comment]
bot, build io.bit3.ThreemaQT

**Comment by tristanlins** [comment]
bot, build io.bit3.ThreemaQT

**Comment by tristanlins** [comment]
bot, build io.bit3.ThreemaQT

File: io.bit3.ThreemaQT.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.14'
+command: ThreemaQT
+finish-args:
+- "--share=ipc"
+- "--socket=x11"
+- "--socket=wayland"
+- "--share=network"
+- "--filesystem=home"
```
Comment (barthalion) [inline io.bit3.ThreemaQT.yml:13]: Does it really need full home access?

File: io.bit3.ThreemaQT.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+app-id: io.bit3.ThreemaQT
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.14'
+command: ThreemaQT
+finish-args:
+- "--share=ipc"
+- "--socket=x11"
```
Comment (barthalion) [inline io.bit3.ThreemaQT.yml:10]: Similarly to WhatsApp one, it should be `fallback-x11` if it works properly on Wayland.

File: io.bit3.ThreemaQT.yml (hunk 1 above)
Comment (tristanlins) [reply io.bit3.ThreemaQT.yml:13]: I almost said it was personal preference. I don't like it when I want to send files and have to move them to Downloads first. And vice versa, if I want to download files, I want to save them directly to a certain location and not in Downloads to move them from there.

But I can switch that to "xdg-downloads" if that's preferred. I can still overwrite the access locally.

File: io.bit3.ThreemaQT.yml (hunk 2 above)
Comment (tristanlins) [reply io.bit3.ThreemaQT.yml:10]: I will change it and try it out.

**Comment by tristanlins** [comment]
bot, build io.bit3.ThreemaQT

**Comment by tristanlins** [comment]
@barthalion I have made the desired changes :-)

**Comment by tristanlins** [comment]
bot, build io.bit3.ThreemaQT

**Comment by tristanlins** [comment]
bot, build io.bit3.ThreemaQT

**Pull request closed without merging**